components/clarinet-deno/index.ts

```diff
@@ -812,8 +813,7 @@ Array.prototype.expectPrintEvent = function (contractIdentifier, value) {
         topic: contract_event.topic,
         value: contract_event.value,
       };
-    } catch (error) {
-      console.warn(error);
```

Comment:
Why removing the log here?

Reply:
If you look at the other `expect` functions, they don't have a log. That's because they're built with the assumption that an error _will_ be thrown if certain properties on the event are undefined.

For example, with a fungible token event, `event.contract_event` is undefined, so calling `event.contract_event.topic` will throw an error. If you have a tx with a FT event and then a log, the warning will be printed, even though it shouldn't be.

Because I added the line at 804, then maybe we don't need the `try/catch` block at all - there should be no other errors. But overall I was just trying to follow the structure of the other functions

Reply:
Alright, makes sense indeed!
Yes I'd like to refacto all these `expect<Event>` method, I think it could be simplified and display better messages in case of failure. Will work on it eventually